webapp/channels/src/components/user_settings/advanced/user_settings_advanced.tsx

```diff
@@ -28,7 +28,8 @@ import PerformanceDebuggingSection from './performance_debugging_section';
 import SettingDesktopHeader from '../headers/setting_desktop_header';
 import SettingMobileHeader from '../headers/setting_mobile_header';
 
-const PreReleaseFeatures = Constants.PRE_RELEASE_FEATURES;
+type PreReleaseFeaturesType = Record<string, {label: string; description: string}>;
```

Comment:
As a future improvement, I think we could probably remove this whole setting. We haven't really used this setting in ages, and the Markdown preview setting was the only one that's used it for years.

Let's not do that in this PR though since it's being backported to 9.5 though

Reply:
created a ticket for this